packages/grid/_modules_/grid/components/GridRow.tsx

```diff
@@ -62,7 +62,7 @@ export const GridRow = (props: GridRowProps) => {
   const rowClassName =
     isFunction(options.getRowClassName) &&
     options.getRowClassName(apiRef!.current.getRowParams(id));
-  const cssClasses = classnames(GRID_ROW_CSS_CLASS, className, rowClassName, {
+  const cssClasses = classnames(GRID_ROW_CSS_CLASS, className, rowClassName, options.classes?.row, {
```

Comment:
I believe that classes should always be defined, and contain the default class name. cc @mnajdova 
```suggestion
  const cssClasses = classnames(className, rowClassName, options.classes.row, {
```

Reply:
One way would be to give it a default option that if the dev passes in a class for the row will be either extended or replaced. This is just for internal management.

Reply:
We have a specific utils that does it in v5, maybe it's something we can use to enforce the same behavior.

Reply:
I will check them, wasn't aware of it, thanks.

Reply:
Updated. The drawback is that `columHeader` is now `colCell` to match the name of the class itself.